book/configuration.md

```diff
@@ -4,7 +4,7 @@
 
 Nushell uses a configuration system that loads+runs two Nushell script files at launch time:
 First, `env.nu`, then `config.nu`.
-Paths to these files can be found by calling `echo $nu.env-path` and `echo $nu.config-path`.
+You can check where Nushell is reading these config files from by running `echo $nu.env-path` and `echo $nu.config-path`.
```

Comment:
In principle you don't need to use `echo` with Nushell (for the most part it is a no-op)

```suggestion
You can check where Nushell is reading these config files from by running `$nu.env-path` and `$nu.config-path`.
```

Reply:
I agree... but I suspect the start of this page is aimed at beginners coming from bash. Should we try to accommodate beginners used to Bash, or get them used to idiomatic Nushell?

Reply:
OK, I updated the PR and I think it's more clear now.